Xamarin.Forms.Core/InputView.cs

```diff
@@ -38,4 +38,10 @@ public bool IsReadOnly
 			set { SetValue(IsReadOnlyProperty, value); }
 		}
 	}
+
+	public enum ClearButtonMode
```

Comment:
Shouldn't the enum, and the property, be named `ClearButtonVisibility` ?

Reply:
@StephaneDelcroix Some reasons I think worth considering:
1. "Visibility" usually makes me think of true/false. This works more like a 'mode'. It depends how you see it I guess.
2. It's easier for developers which are used with iOS API to instantly know what is it about: https://developer.apple.com/documentation/uikit/uitextfieldviewmode?language=objc
3. ClearButtonMode is shorter than ClearButtonVisibility.

Reply:
Obviously these are suggestions, I don't emotionally attach to this stuff usually, you (team) make the  last call

Reply:
Visibility makes more sense to me. We have many other uses of Visibility that are mapped to enum values, too.

Reply:
OK, I renamed to `CloseButtonVisibility`, see the commit